Apps/WebClient/src/ClientApp/src/models/userRating.ts

```diff
@@ -0,0 +1,7 @@
+export default interface UserRating {
```

Comment:
Should we capture date & time of submission? if yes, need a date & time as the dbmodel.